package.json

```diff
@@ -6,9 +6,7 @@
   "license": "UNLICENSED",
   "private": true,
   "scripts": {
-    "ubuntu": "chmod +x ./scripts/ubuntu.sh && ./scripts/ubuntu.sh",
-    "windows": "chmod +x ./scripts/ubuntu.sh && ./scripts/ubuntu.sh",
-    "macosx": "chmod +x ./scripts/macosx.sh && ./scripts/macosx.sh",
+    "preinstall": "chmod +x install.sh && ./install.sh",
```

Comment:
Excellent! Let's update the README to reflect that the command is just `yarn install && yarn start` now.

Reply:
Changes done.